### DISCUSSION TITLE
Webring sdy - 60 2024

### DISCUSSION THREAD

**Discussion by Marigian** [sdy60-2024/sdy60#3] [Show and tell]
Καλησπέρα, 

Εχω βρεί ένα τρόπο που μπορούμε να υλοποιήσουμε το webring που είναι το ζητούμενο για την ομαδική στο Μερος Α. 

Προς το παρόν έχω κάνει το σετάρισμα και φαίνεται να δουλεύει σωστά θα θέλατε να ρίξετε μια ματιά και να δούμε αν θα προχωρήσουμε με αυτή την υλοποίηση? Το αποθετήριο είναι αυτό **https://github.com/sdy60-2024/webring** είναι public οπότε λογικά το βλέπετε όλοι και έχετε πρόσβαση. 
Είναι σχετικά απλό καθώς χρειάζεται απο τη πλευρά των συμμετεχόντων να γίνει τροποποίηση σε δύο κομμάτια στα αρχεία του κώδικα, έχω γράψει και σχετικές οδηγίες.

**Comment by Pagiatis-Spiros** [comment]
Μαρίνα σε ευχαριστούμε για την δουλειά που κάνεις, εγώ δεν είχα πάρει χαμπάρι όπου άνοιξες φόρουμ εδώ και έτσι δεν τα είχα δει αυτά.

Σε συνέχεια αυτού που είπε ο κ.Χωριανόπουλος (δηλ. να βάλουμε link από το βιογραφικό μας στην homepage), εγώ το έκανα βάζοντας το παρακάτω link στο τέλος του βιογραφικού μου.

\#\# \[Back to homepage](https://sdy60-2024.github.io/webring/)
